packages/main/src/List.js

```diff
@@ -179,6 +179,37 @@ const metadata = {
 			},
 		},
 
+		/**
```

Comment:
Have you considered having a new list? Or it's an overkill? I'm fine with this solution, this will be very minimal when code is minified.

Reply:
Yes I thought about this, as these events are specific to notifications, at the end I decided that it not so much code in addition.